package.json

```diff
@@ -56,17 +58,22 @@
     "eslint-watch": "^6.0.1",
     "jest": "^26.0.1",
     "jsdoc-to-markdown": "^6.0.1",
+    "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.0",
+    "react-router-dom": "^5.2.0",
     "react-test-renderer": "^16.13.1"
   },
   "dependencies": {
+    "@material-ui/icons": "^4.9.1",
```

Comment:
make this a `peerDependency` (and include in `devDependencies` for storybook and local development)